activity_hub.php

```diff
@@ -140,17 +140,26 @@
 
 // ----------------------------------
 
+/**
+ * Print out the snapshot table.
+ *
+ * The table contains a row for each stage including the user's access for
+ * that stage as well as project and page metrics.
+ *
+ * @param bool $show_filtered_projects
+ *   If TRUE, the table will show numbers based on the user's project filter
+ *   for that stage.
+ *
+ * @param bool $show_filtering_links
+ *   If TRUE, the table will allow the user to toggle between viewing numbers
+ *   for ALL projects and just those for filters.
+ *
+ * @param bool $show_beginner_help
+ *   If TRUE, the table will be prefaced with an informational paragraph.
+ *
+ * @return void
+ */
 function progress_snapshot_table($show_filtered_projects, $show_filtering_links, $show_beginner_help)
```

Comment:
should we start using php type declarations too since we're specifying types in the phpdocs now? I realize this changes runtime behavior with TypeError, so maybe safer to not. Just an idea.

Reply:
I think so, yes. We'll have to do it gradually and intentionally but we're at a point where that makes sense with our minimum PHP version, etc.